bve/src/parse/mesh/mod.rs

```diff
@@ -17,51 +25,73 @@ pub mod instructions;
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq)]
 pub enum FileType {
+    /// No comma after instruction name
     B3D,
+    /// Comma after instruction name
     CSV,
 }
 
+/// A single static object.
+///
+/// Despite having many meshes, the game treats this as a single object, and cannot be toyed
+/// with at a more granular level.
```

Comment:
Why does it have many meshes? If it has many meshes, why can't I toy with them individually? Is this just some legacy bullshit?

Reply:
The game treats it as a single mesh for placement purposes, you can do whatever you please with the individual ones. They are separate because they have various per-mesh properties.

Reply:
I guess I'm not clear on what "toyed-with" means in this context